Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Resources.EditorWindow
+{
+    public class GameLiftPlugin : UnityEditor.EditorWindow
+    {
+        [SerializeField] private Texture icon;
+
+        private VisualTreeAsset _mVisualTreeAsset;
```

Comment:
Why is this named differently than other member variables i.e the `_m` prefix

Reply:
Renamed